docs/build.md

```diff
@@ -0,0 +1,61 @@
+# Installation
+
+## Installation using Spack
+
+The simplest way to install ArborX is to use [Spack](https://github.com/spack/spack) package manager:
+```shell
+spack install arborx
+```
+
+## Manual installation
+
+ArborX has the following dependencies:
+
+Dependency | Version | Required
+--- | --- | ---
+Kokkos | 2.8.00 | :heavy_check_mark:
+MPI    | 2      |
+CMake  | 3.14   | :heavy_check_mark:
```

Comment:
I would also add `Boost` and `benchmark` even if it's just for testing and examples.

Reply:
Examples do not need `boost`. Benchmarks are not for users, I think. So I would argue against complicating this.

Reply:
Question to all: if we don't use boost for core and examples, what is the minimum version of CMake that we need?

Reply:
I don't know but we only require 3.12 in any case

Reply:
Also we may use a different marker to signify required for unit tests.

Reply:
@dalg24 1. Where is 3.12 requirement coming from? 2. I don't think we need to talk about boost at all.

Reply:
https://github.com/arborx/ArborX/blob/d41aabc39a0cacc00c9ef9a44f85f145b3c796ba/CMakeLists.txt#L1